src/main.js

```diff
@@ -93,6 +94,9 @@ function TimekitBooking() {
       // Render available timeslots in FullCalendar
       if(response.data.length > 0) renderCalendarEvents(response.data);
 
+      // Render test ribbon if enabled 
+      if (response.headers['timekit-testmode']) renderTestModeRibbon();
```

Comment:
Shouldn't this be in PascalCase too? I'd be surprised if tests pass as object keys are case sensitive. Maybe Axios converts them to lowercase automatically?

Reply:
yes, they will fail with Pasal case

Reply:
OK, seems like Axios indeed converts them: https://github.com/mzabriskie/axios/issues/413